cmd/analyzer/parse_args.go

```diff
@@ -210,8 +210,15 @@ func errorInErgs(args *InArgs, flagset *flag.FlagSet) error {
 	return nil
 }
 
+//gocyclo:ignore
 func notSupportedYetArgs(args *InArgs) error {
 	diffAnalysis := *args.AnalysisType == allEndpointsDiff || *args.AnalysisType == allSubnetsDiff
+	if *args.OutputFormat == DRAWIOFormat || *args.OutputFormat == ARCHDRAWIOFormat {
+		if *args.AnalysisType != allEndpoints && *args.AnalysisType != allSubnets {
+			return fmt.Errorf("drawio output format is not supported with %s analysis type", *args.AnalysisType)
+		}
+		return nil
+	}
```

Comment:
Why was this check added now?

Reply:
yes, its not related, it was needed to be added when adding diff analysis. and\or signal subnet